integrations/slack/events/app_home_opened.go

```diff
@@ -0,0 +1,39 @@
+package events
```

Comment:
I tried to create a file for each group of events, not for each event. Please move this into `app_mention.go` and rename the file `app.go`